vdb/lib/__init__.py

```diff
@@ -391,6 +429,23 @@ def __init__(
         self.confidentiality_impact = confidentiality_impact
         self.integrity_impact = integrity_impact
         self.availability_impact = availability_impact
+        self.vector_string = vector_string
```

Comment:
Is this vector string different from attack_vector?

Reply:
@prabhu Yes, attack vector is an enum like NETWORK, PHYSICAL, etc. while the vector string is something like CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H. The [vector string](https://www.first.org/cvss/specification-document#Vector-String) is a required component to give a cvss_v3 score in CSAF. [CSAF 2.0](https://github.com/oasis-tcs/csaf/blob/master/csaf_2.0/json_schema/csaf_json_schema.json) validates this by specifying the use of either the cvss [v3.0 ](https://www.first.org/cvss/cvss-v3.0.json)or [v3.1](https://www.first.org/cvss/cvss-v3.1.json) schemas.